src/test/java/co/rsk/bitcoinj/script/ScriptTest.java

```diff
@@ -505,42 +508,47 @@ public void numberBuilderNegative() {
     }
 
     @Test
-    public void getNumberOfSignaturesRequiredToSpend_fast_bridge_redeem_script() {
-        byte[] flyoverDerivationHash = Sha256Hash.of(new byte[]{1}).getBytes();
-        Script fastBridgeRedeemScript = RedeemScriptUtils.createFastBridgeRedeemScript(
-            flyoverDerivationHash, FEDERATION_KEYS);
-        int actualNumberOfSignaturesRequiredToSpend = fastBridgeRedeemScript.getNumberOfSignaturesRequiredToSpend();
+    public void getNumberOfSignaturesRequiredToSpend_whenFlyoverRedeemScript_ok() {
```

Comment:
WDYT about improving the test name by adding "shouldReturn..."?

Reply:
Updated.